packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -799,9 +835,150 @@ export const RemoteServerFactory = {
+        }) {
+            return URL.format({
+                pathname: path.replace(replaceRegex, ''),
+                query
+            })
+        }
+
+        function getCommaDelimitedHeaders({headersMap, separator = ',', lowerCaseKey = false}) {
+            const commaDelimitedHeaders = {}
+
+            Object.entries(headersMap).forEach(([headerKey, headerValue]) => {
+                const newKey = lowerCaseKey ? headerKey.toLowerCase() : headerKey
+                if (Array.isArray(headerValue)) {
+                    commaDelimitedHeaders[newKey] = headerValue.join(separator)
+                } else {
+                    commaDelimitedHeaders[newKey] = headerValue
+                }
+            })
+
+            return commaDelimitedHeaders
+        }
+
+        function getEventBody({event, body = event.body, isBase64Encoded = event.isBase64Encoded}) {
+            return Buffer.from(body, isBase64Encoded ? 'base64' : 'utf8')
+        }
+
+        const _serverlessExpressHandler = serverlessExpress({
+            app,
+            // the underlying serverless-express library strips `headers`
+            // which our ssr-infrastructure expects, these mappings restore
+            // and pass through all request / response properties
+            eventSource: {
+                getRequest: ({
+                    event,
+                    method = event.httpMethod,
+                    path = getPathWithQueryStringParams({event})
+                }) => {
+                    let headers = {}
+
+                    if (event.multiValueHeaders) {
+                        headers = getCommaDelimitedHeaders({
+                            headersMap: event.multiValueHeaders,
+                            lowerCaseKey: true
+                        })
+                    } else if (event.headers) {
+                        headers = event.headers
+                    }
+
+                    let body
+
+                    if (event.body) {
+                        body = getEventBody({event})
+                        const {isBase64Encoded} = event
+                        headers['content-length'] = Buffer.byteLength(
+                            body,
+                            isBase64Encoded ? 'base64' : 'utf8'
+                        )
+                    }
+
+                    const remoteAddress =
+                        (event &&
+                            event.requestContext &&
+                            event.requestContext.identity &&
+                            event.requestContext.identity.sourceIp) ||
+                        ''
+
+                    return {
+                        method,
+                        headers,
+                        body,
+                        remoteAddress,
+                        path
+                    }
+                },
+                getResponse: ({statusCode, body, headers, isBase64Encoded}) => {
```

Comment:
@bfeister I don't see this `getResponse` parameter defined in the library code https://github.com/vendia/serverless-express/blob/mainline/src/configure.js#L18-L29

Could you explain further how it is used?